app/sim_test.go

```diff
@@ -173,7 +169,7 @@ func TestAppStateDeterminism(t *testing.T) {
 
 			fmt.Printf(
 				"running non-determinism simulation; seed %d: %d/%d, attempt: %d/%d\n",
-				config.Seed, i+1, numSeeds, j+1, numTimesToRunPerSeed,
+				config.Seed, i+one, numSeeds, j+one, numTimesToRunPerSeed,
```

Comment:
You don't need variable declaration for this. You can avoid golangci warning here
```suggestion
				config.Seed, i+1, numSeeds, j+1, numTimesToRunPerSeed,
```

Reply:
@akhilkumarpilli can you address this also. Remove variable usage for this particular case.